collector/src/lib.rs

```diff
@@ -822,7 +823,7 @@ mod tests {
         let collector = setup_collector(vdaf);
 
         let batch_interval = Interval::new(
-            Time::from_seconds_since_epoch(1_000_000),
+            Time::from_seconds_since_epoch(2_000_000), // TODO why???
```

Comment:
Not sure offhand -- if you don't know why this change is necessary, I think it's OK to just drop the TODO.

Reply:
I saw a test flake in this module yesterday, so it was probably random chance, unconnected to this timestamp. (I suspect we may be running into a `backoff` timer we need to tweak, but I haven't had a chance to dig into it yet)